src/pages/index.js

```diff
@@ -92,36 +92,35 @@ export default ({ ...data }) => {
             <Notification />
             <Hero>
                 <Heading>
-                    {`ideas.find({"attributes" : ["fast", "innovative",
-                    "original"]})`}
+                    {`ideas.find({"attributes":`}
+                    <br />
+                    {`["fast", "innovative", "original"]})`}
                 </Heading>
                 <Sub>What will you create today?</Sub>
                 <CardGallery>
-                    <StyledTopCard image={unityImage}>
+                    <StyledTopCard image={unityImage} to="/article">
```

Comment:
Similarly, should these go to the learn page? Looks like `/article` has moved so it 404s for me.

Reply:
They'll go to specific articles